installer/dockerfile/admission/Dockerfile

```diff
@@ -15,5 +15,16 @@
 
 FROM alpine:latest
 
+# Install requirements
```

Comment:
the file name should be `Dockerfil.secret`, and only include `gen-admission-secret.sh`.

Reply:
The sh file and binary are kept in one container in default since they are both simple logic and work for admission service, adding one more image can be complicated for maintain and the usage of end user.

Reply:
> adding one more image can be complicated for maintain and the usage of end user

If so, why `Dockerfile.only_binay` ?

Reply:
ok, will remove this.